dev-python/dukpy/dukpy-0.3.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="JavaScript in Python"
+DESCRIPTION="dukpy is a JavaScript runtime environment for Python (tested with python 2.7 and >= 3.4) using the duktape embeddable JavaScript engine. With dukpy, you can run JavaScript in Python."
+HOMEPAGE="https://github.com/kovidgoyal/dukpyt"
+COPYRIGHT="Duktape authors"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/kovidgoyal/dukpy/archive/v0.3.tar.gz"
+CHECKSUM_SHA256="cc78c39ebba51f381c623b164cfb7dcf3caddf515fe7094bc53b7eca5d4e435e"
+
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+	cmd:gcc"
+PYTHON_PACKAGES=(python36 python3)
```

Comment:
Move the " to the next line after a tab, insert a new line between this block and the next one:
```diff
-	cmd:gcc"
-PYTHON_PACKAGES=(python36 python3)
+	cmd:gcc
+	"
+
PYTHON_PACKAGES=(python36 python3)

Reply:
We need cmg:gcc here?

Reply:
Left a missing remark, think it should be clear now

Reply:
like this?